scripts/install.sh

```diff
@@ -60,3 +60,15 @@ cp bin/otel-cli /usr/local/bin/
 chmod a+rx /usr/local/bin
 
 systemctl reload ssh
+
+# hardcode /etc/hosts entries so we don't need DNS
+
+if grep -q "## start opensafely dns" /etc/hosts; then
+    # -z use NULL for new line, enables multiline matching
+    # first -e adds the current file content
+    # second -e removes the old file content
+    sed -z -e '/## start opensafely dns.*## end opensafely dns/r etc/opensafely/hosts' -e 's/## start opensafely dns.*## end opensafely dns//' -i.bak hosts
+else
```

Comment:
do we really need this? Could we simplify & just put a note in `etc/opensafely/hosts` saying "this file is managed automatically, do not edit"?

Reply:
hmmmm yeah but the defaults

Reply:
right, that's why I kept it

Reply:
my pref would be putting the ubuntu default in something like `mybackend:/etc/hosts.default` or even `backend-server/etc/hosts` & concatting two files every time rather than attempting to `sed` the files, but I know you're going to hate that!

Reply:
I don't hate it, I'm just wary that the ubuntu default may be manipulated by other things

Reply:
well let's do it this way for now, we can always regret it later if we want.